### PR TITLE
trivial: Add some additional docs about CCGX factory mode

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [X] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@superm1 approved by Infineon -- can you do a quick grammar check please.

**Review by superm1 — APPROVED** [review]

File: plugins/ccgx/README.md [hunk 1]
```diff
@@ -56,6 +56,36 @@ The `CY_PD_JUMP_TO_ALT_FW_CMD_SIG` command is allowed only in asymmetric FW, but
 In composite firmware topology, a single firmware image contains metadata and
 firmware images of multiple devices including DMC itself in a dock system.
 
+### DMC Factory Mode
+
+Dock Management Controller devices have a *composite version* that is used to
+describe the dock hardware as a whole, rather than enumerating and updating
+each sub-component separately.
+In some cases, typically when OEMs have not followed the IHV-approved factory
```
[the remark this answers is not in the document]
Comment (hughsie) [reply plugins/ccgx/README.md:64]: What about `When OEMs have not followed the IHV-approved factory...`

[the remark this answers is not in the document]
Comment (superm1) [reply plugins/ccgx/README.md:64]: That's good

[the remark this answers is not in the document]
Comment (hughsie) [reply plugins/ccgx/README.md]: Fixed.

**Pull request merged**